notebooks/WFC3/ir_scattered_light_calwf3_corrections/Correcting_for_Scattered_Light_in_IR_Exposures_Using_calwf3_to_Mask_Bad_Reads.ipynb

```diff
@@ -117,7 +117,7 @@
     "\n",
```

Comment:
<p>Hi @annierose3, The code is nicely structured and achieves its objectives efficiently. To enhance it further, I suggest incorporating context managers for file operations, like this: <code>with fits.open(reprocessed_flt, mode='update') as image_new, fits.open(original_ima) as ima_orig:</code></p><p>.Additionally, adding explanatory comments would greatly improve its readability and help others understand the logic more easily. These adjustments would elevate the code's clarity and effectiveness. Thanks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>